community/pom.xml

```
@@ -35,10 +35,9 @@
         <url>https://github.com/bonitasoft/bonita-admin-application</url>
     </scm>
 
-
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <application.version>6.4</application.version>
+        <application.version>6.5</application.version>
```

```suggestion
        <application.version>6.4</application.version>
```
We already bumped the version for the next version